test/inductor/test_flex_attention.py

```diff
@@ -2892,80 +2974,14 @@ def test_kernel_options_argument_is_respected(self):
 
         # Ensure we respect user's input kernel options.
         _, code = run_and_get_code(
-            torch.compile(flex_attention), q, k, v, kernel_options={"BLOCK_M": 16}
+            torch.compile(flex_attention, fullgraph=True),
+            q,
+            k,
+            v,
+            kernel_options={"BLOCK_M": 16},
         )
         FileCheck().check("BLOCK_M : tl.constexpr = 16").run(code[0])
 
-    @supported_platform
-    @unittest.skipIf(SKIP_UT_ON_CPU, "Skip on CPU as not supported")
-    def test_comparison_vs_sdpa(self):
```

Comment:
why remove this test?

Reply:
It should have been there after I made the new test class, so while Im here might as well make it better :)

Reply:
Super old test that was no longer relevant IMO